### ISSUE TITLE
High-Performance Face Recognition Library on PyTorch

### ISSUE THREAD

**Issue by vvoovv** [vvoovv/notes#94]
* [face.evoLVe](https://github.com/ZhaoJ9014/face.evoLVe.PyTorch) by [Jian Zhao](https://github.com/ZhaoJ9014/face.evoLVe.PyTorch)

This repo provides a comprehensive face recognition library for face related analytics & applications, including face alignment (detection, landmark localization, affine transformation, etc.), data processing (e.g., augmentation, data balancing, normalization, etc.), various backbones (e.g., ResNet, IR, IR-SE, ResNeXt, SE-ResNeXt, DenseNet, LightCNN, MobileNet, ShuffleNet, DPN, etc.), various losses (e.g., Softmax, Focal, Center, SphereFace, CosFace, AmSoftmax, ArcFace, Triplet, etc.) and bags of tricks for improving performance (e.g., training refinements, model tweaks, knowledge distillation, etc.).
